### DISCUSSION TITLE
Running windows commands within podman machine distribution

### DISCUSSION THREAD

**Discussion by mwoodpatrick** [containers/podman#14564] [Q&A]
In my other WSL distributions I can run a command like:

    /mnt/c/WINDOWS/explorer.exe

Which will bring up the explorer app and show me my current directory.

However, when I run this command in the podman wsl distribution I get

     <3>init: (1165) ERROR: UtilAcceptVsock:245: accept4 failed 110

How can I get this to work in the podman WSL distro?

**Comment by n1hility** [comment]
@mwoodpatrick this sounds like: https://github.com/microsoft/WSL/issues/7947
